### PR TITLE
StatefulSet and parallel management

### PR DESCRIPTION
**Pull request by DOCtorActoAntohich**
idk what to say here, just look at my cat
image

### CODE REVIEW

**Comment by Andrew174194** [comment]
> idk what to say here, just look at my cat !

But that is not your hands... 🤔

File: k8s/make-your-time/templates/statefulset.yaml [hunk 1]
```diff
@@ -59,18 +60,18 @@ spec:
...
+          #     path: /
+          #     port: {{ .Values.service.container_port }}
+          #   initialDelaySeconds: 10
+          #   periodSeconds: 10
+          # readinessProbe:
+          #   httpGet:
+          #     path: /
+          #     port: {{ .Values.service.container_port }}
+          #   initialDelaySeconds: 10
+          #   periodSeconds: 10
```
Comment (Andrew174194) [inline k8s/make-your-time/templates/statefulset.yaml:74]: You could set the parameter in `values.yaml` to enable/disable this stuff... ¯\_(ツ)_/¯

File: k8s/StatefulSet.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+pod/make-your-time-1   0/1     ContainerCreating   0          1s
+```
+
+Third call:
+
+```
+NAME                   READY   STATUS              RESTARTS   AGE
+pod/make-your-time-0   1/1     Running             0          3s
+pod/make-your-time-1   1/1     Running             0          2s
+pod/make-your-time-2   0/1     ContainerCreating   0          1s
```
Comment (Andrew174194) [inline k8s/StatefulSet.md:111]: Are AGEs real? For me it takes 5 sec. between new pod creations...

Comment (DOCtorActoAntohich) [reply k8s/StatefulSet.md:111]: skill issue.

yes, they be real

File: k8s/make-your-time/templates/statefulset.yaml (hunk 1 above)
Comment (DOCtorActoAntohich) [reply k8s/make-your-time/templates/statefulset.yaml:74]: didn't bother checking tbh. even though that amongus-like behavior i just wanted to speedrun this one as soon as ASAP

**Comment by justmark0** [comment]
кот крут!! и твой мр тоже!!
eng: gud

**Pull request merged**